lib/components/narrative/line-itin/line-itinerary.js

```diff
@@ -74,6 +75,9 @@ export default class LineItinerary extends NarrativeItinerary {
           timeOptions={timeOptions}
           onClick={onClick}
         />
+
+        <span className='pull-right'><LinkButton to='/savetrip'>Save this option</LinkButton></span>
```

Comment:
Once a trip is saved, there is no indication that the trip (itinerary) has been saved. It would be nice to have the itinerary display change accordingly once it is saved.